geoadmin/src/main/java/org/activityinfo/geoadmin/merge2/view/swing/match/select/SyncingColumnListener.java

```diff
@@ -0,0 +1,53 @@
+package org.activityinfo.geoadmin.merge2.view.swing.match.select;
+
+import javax.swing.*;
+import javax.swing.event.ChangeEvent;
+import javax.swing.event.ListSelectionEvent;
+import javax.swing.event.TableColumnModelEvent;
+import javax.swing.event.TableColumnModelListener;
+import javax.swing.table.TableColumn;
+
+/**
+* Keeps a header table's column sizes in sync with a body table below.
+*/
+class SyncingColumnListener implements TableColumnModelListener {
+
+    private JTable headerTable;
+    private JTable bodyTable;
+
+    public SyncingColumnListener(JTable headerTable, JTable bodyTable) {
+        this.headerTable = headerTable;
+        this.bodyTable = bodyTable;
+    }
+
+    @Override
+    public void columnAdded(TableColumnModelEvent e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)